src/context/AuthContext.js

```diff
@@ -62,7 +62,7 @@ const AuthProvider = ({ children }) => {
           type: "SAVE_USER_INFO",
           payload: foundUser,
         });
-        navigate("/");
+        navigate(-1);
```

Comment:
why navigate is -1 ??

Reply:
If user visits the cart route & is not logged in. So user will click on login button & navigate to login page. After logging in, user will navigate back to cart page. 
User will navigate back to the same page where he/she was before logging in. ( UX stuff )